packages/webpack/src/plugins/wp5-fix.js

```diff
@@ -0,0 +1,12 @@
+module.exports = class WP5Fix {
```

Comment:
Can be removed?

Reply:
Yes actually it was to hide deprecated warnings but we should indeed try to update any problematic 3rd party plugin instead of hiding :) Will remove

Reply:
Done